bin/sozo/src/commands/migrate.rs

```diff
@@ -22,44 +22,29 @@ use super::options::world::WorldOptions;
 pub struct MigrateArgs {
     #[command(subcommand)]
     pub command: MigrateCommand,
+
+    #[arg(long, global = true)]
+    #[arg(help = "Name of the World.")]
+    #[arg(long_help = "Name of the World. It's hash will be used as a salt when deploying the \
```

Comment:
We may add the info that if `name` is not provider, the package id will be used instead?